### PR TITLE
Review my tribute repo

### PR DESCRIPTION
**Pull request by mulugithub**
### The following requirements are checked while reviewing the code:

- Does the reviewed project run on your machine?
- Does the reviewed project work properly?
- Are there any [linters errors](https://github.com/microverseinc/linters-config)?
- Is the correct flow used?
- Is the repository [documented in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)?
- Are there any bugs that were missed by the author?
- Is there any room for improvement in the project?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# Tribute-website
\ No newline at end of file
+# Tribute_Repo
+This is a website that allows well wishers write some heartfelt tribute to a late friend.
```
Comment (mulugithub) [inline README.md:2]: Please make the readme file more descriptive and professional.  I would suggest you check out this [Microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and customize it accordingly.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+ <main id="main">
+    <h1 id="title">Dr. Norman Borlaug"</h1>
+    <span id="title-description">The man who saved a billion lives</span>
```
Comment (mulugithub) [inline index.html:13]: I would suggest you use a header tag before the main tag and put lines 12 and 13 inside the header tag for good semantics and best practices.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+
+    </div>
+    <div>
+        <p><i>
+            "Borlaug's life and achievement are testimony to the far-reaching contribution that one man's towering intellect, persistence and scientific vision can make to human peace and progress."
+
+           <br> -- Indian Prime Minister Manmohan Singh</p>
+        </i>
+    </div>
```
Comment (mulugithub) [inline index.html:46]: I would suggest you use the main tag for those content for best practice.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    <div id="tribute-link">
+        <h3><strong>
+            If you have time, you should read more about this incredible human being on his
+        </strong></h3>
+        <a href="https://en.wikipedia.org/wiki/Norman_Borlaug"  target="_blank"> Wikipedia entry</a>
+    </div>
```
Comment (mulugithub) [inline index.html:52]: Kindly write the footer of the page under the footer tag after the main tag for good semantics.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="index.css">
```
Comment (mulugithub) [inline index.html:8]: Please include your external CSS file with the correct file name. To do this, replace index.css with styles.css.

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,43 @@
+body {
+    width: 100%;
+    padding: 20px;
+    margin: 0px
+}
```
Comment (mulugithub) [inline styles.css:5]: Please make a change expected semicolon here. I would recommend you add a semicolon margin: 0px;

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    padding: auto;
+    background-color: white;
+}
+#img-div img {
+    position: relative;
+
+    width: 70%;
+    height: 100%;
+    margin: auto;
+}
```
Comment (mulugithub) [inline styles.css:38]: Kindly add a new line before the CSS rule and Expected two space indentation after the selector.  it violates the linter rule.

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+div figcaption {
+    margin-top: 30px;
+    margin-bottom: 30px;
+}
+
```
Comment (mulugithub) [inline styles.css:43]: Not expected more than one empty space. please remove the empty line.

**Review by mulugithub — COMMENTED** [review]
# Required Changes ♻️ 
Check the comments under the review
# Optional suggestions
_Every comment with [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better_


Happy coding and cheers 😄

**Review by Bantealem — APPROVED** [review]
Hi @mulugithub 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- [x]  Correct GitHub README.
- [x] Polite communication.
- [x] Great inline comments.
## Optional suggestions
- [ ] You did in requesting changes for the code. Kindly, add more explanation why you suggested changes. It inspires the student to get a better knowledge of that specific change.
- [ ] Kindly use emojis and gifs. It makes your review friendly. 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**